toolkit/scripts/README.md

```diff
@@ -0,0 +1,9 @@
+# Scripts
```

Comment:
I'd also see, if we can link to this new doc from the existing ones, so it's easier to find.

Reply:
I don't think we refer to the scripts folder within other doc files of this repo? I was going to add a link to this file on our wiki page and update the pages to refer to it.

Reply:
In that case I'd pick one of the two ways:
- Either add the link to this readme from the dev-oriented ones already "reachable" from the main readme or
- remove this readme completely and rely only on the internal wiki.

I'm fine either way, if we never mentioned the scripts in GitHub documentation before.

Reply:
Moved it to a wiki page https://dev.azure.com/mariner-org/mariner/_wiki/wikis/mariner.wiki/1591/CBL-Mariner-scripts